staging/src/k8s.io/sample-apiserver/pkg/registry/registry.go

```diff
@@ -31,7 +31,7 @@ type REST struct {
 // RESTInPeace is just a simple function that panics on error.
 // Otherwise returns the given storage object. It is meant to be
 // a wrapper for wardle registries.
-func RESTInPeace(storage rest.StandardStorage, err error) rest.StandardStorage {
+func RESTInPeace(storage rest.Storage, err error) rest.Storage {
```

Comment:
I can't remember, do we have a test that actually tries to create a flunder?  I'm never sure whether passing through a limiting cast like this confuses future type assertions into not realizing that this a rest.Getter as a for instance.

Reply:
> do we have a test that actually tries to create a flunder?

No. Related https://github.com/kubernetes/sample-apiserver/issues/7.

I'll add integration tests for this.